src/Simplex_tree/benchmark/random_simplices.h

```diff
@@ -0,0 +1,38 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2025 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <iostream>
+#include <random>
+#include <numeric>  // for std::iota
+#include <algorithm>  // for std::shuffle
+#include <vector>
+
+std::random_device rd;
+
+template <class Vertex_handle>
+std::vector<Vertex_handle> rand_int_range(int subset_min_size,
```

Comment:
Add a comment above explaining what this computes? (or give the function a name that makes it obvious, `rand_int_range` doesn't)

Reply:
I renamed `rand_int_range` as `random_simplex` and added some internal doc on [958156b](https://github.com/GUDHI/gudhi-devel/pull/1229/commits/958156bae11386ec2b9a8f2104115cd18d221711)